### ISSUE TITLE
osc no longer works with OBS 2.6

### ISSUE THREAD

**Issue by jengelh** [openSUSE/osc#280]
When running osc-0.157.1-1.1 (openSUSE Tumbleweed) against a somewhat older OBS Server:
```
10:36 a4:../core_master/kopano > osc -d r
GET https://master.zbuild.net/build/core:master/_result?multibuild=1&locallink=1&package=kopano
Server returned an error: HTTP Error 400: Bad Request
Date: Tue, 21 Mar 2017 09:37:10 GMT
Server: Apache
Cache-Control: private
Vary: Accept-Encoding
X-XSS-Protection: 1; mode=block
X-Opensuse-Runtimes: {"view":null,"db":0.9885390000000001,"backend":0,"xml":0}
X-Request-Id: 7ed1dc97-ea95-4627-988c-a1c6707dddf8
X-Opensuse-APIVersion: 2.6.51.git20160421.74ddcbe
Content-Disposition: attachment
Content-Transfer-Encoding: binary
X-Runtime: 0.022701
X-Frame-Options: SAMEORIGIN
X-Content-Type-Options: nosniff
X-Powered-By: Phusion Passenger 5.0.22
Status: 400 Bad Request
Cache-Control: public
Connection: close
Transfer-Encoding: chunked
Content-Type: text/xml

<status code="400">
  <summary>unknown parameter 'locallink', 'multibuild'</summary>
</status>

unknown parameter 'locallink', 'multibuild'
```

**Comment by lethliel** [comment]
please use the `--no-multibuild` switch. The default behavior was changed with PR #257.

A compatibility mode for old APIs as introduced in https://github.com/openSUSE/osc/pull/276/commits/e94c9cc280ac788705bac12ba674e28bfb9a7978 and will be released with 0.157.2 soon.

**Comment by lethliel** [comment]
0.157.2 was released.
